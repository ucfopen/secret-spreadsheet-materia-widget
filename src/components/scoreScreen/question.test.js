import React from 'react';
import Question from './question';
import renderer from 'react-test-renderer';

describe('Question', () => {
	test('Rendered with question description', () => {
		const props = {
			question: 'Test Question',
			description: 'Test description'
		}

		const component = renderer.create(<Question {... props} />);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});

	test('Rendered without question description', () => {
		const props = {
			question: 'Test Question',
			description: ''
		}

		const component = renderer.create(<Question {... props} />);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
