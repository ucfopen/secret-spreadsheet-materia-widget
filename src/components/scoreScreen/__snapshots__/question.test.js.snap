// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question Rendered with question description 1`] = `
<header>
  <h3>
    Test Question
  </h3>
  <p>
    Test description
  </p>
</header>
`;

exports[`Question Rendered without question description 1`] = `
<header>
  <h3>
    Test Question
  </h3>
</header>
`;
